website/server/controllers/api-v3/groups.js

```diff
@@ -480,8 +480,7 @@ api.removeGroupMember = {
 
     let validationErrors = req.validationErrors();
     if (validationErrors) throw validationErrors;
-
-    let optionalMembership = user.contributor.admin ? true : false;
+    let optionalMembership = user.contributor.admin;
```

Comment:
Thanks, I had also missed one of the other ternaries in this commit so I'll fix this and take into account your suggestion.